packages/cli/src/build.ts

```diff
@@ -366,12 +366,12 @@ export function build({
     undefined,
     {
       before: [
+        ...getTransformers(type, options),
```

Comment:
Why are we changing the order here?

Reply:
This was an attempt to work around the resolution logic flaw, but I ended up fixing that instead. Reverted this now.